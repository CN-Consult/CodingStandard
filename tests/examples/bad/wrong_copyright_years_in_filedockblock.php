<?php
/**
 *
 * @file
 * @version 1.0
 * @copyright 2020-2017 CN-Consult GmbH
 * @author Daniel Haas <daniel.haas@cn-consult.eu>
 */
 