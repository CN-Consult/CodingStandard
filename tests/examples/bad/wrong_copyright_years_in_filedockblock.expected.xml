<?xml version="1.0" encoding="UTF-8"?>
<phpcs version="does not matter">
    <file name="examples/bad/wrong_copyright_years_in_filedockblock.php" errors="0" warnings="1" fixable="0">
        <warning line="6" column="4" source="..FileComment.InvalidCopyright" severity="5" fixable="0">Invalid year span "2020-2017" found; consider "2017-2020" instead</warning>
    </file>
</phpcs>
